src/components/Pagination/Pagination.tsx

```diff
@@ -2,0 +1,74 @@
+import React from 'react';
+import cn from 'classnames';
+
+interface PaginationProps {
+  pageItemsToShow: number[];
+  pageItemCount: number;
+  currentPage: number;
+  visibleItems: string[];
```

Comment:
The `visibleItems` prop is not necessary for the `Pagination` component according to the task requirements. Consider removing it to align with the specified props.